core-blocks/image/edit.js

```diff
@@ -122,7 +126,26 @@ class ImageEdit extends Component {
 		} );
 	}
 
-	onSetHref( value ) {
+	onSetLinkDestination( value ) {
+		let href;
+
+		if ( value === LINK_DESTINATION_NONE ) {
+			href = '';
```

Comment:
Is there a reason we want this to be an empty string instead of `null`? `null` seems more appropriate.

Also, I'd just set the "none" value as the default here; start with `let href = '';` (or `let href = null;` and then change it if the link destination is something else.

Reply:
Adding `default: 'none'` to the attribute definition itself feels right to me in this case.

Reply:
Yeah that's fine too. Just something over the way it's being done now would be good.

Reply:
I've added `default: 'none'` 👍 

Also added some extra test fixtures to the full-content test, so that's a little bit of coverage.